# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sendmail/setmailer.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- sendmail-8.14.0/sendmail/Makefile.m4.orig	2007-01-09 00:04:09.000000000 +0000
+++ sendmail-8.14.0/sendmail/Makefile.m4	2007-02-10 11:32:24.000000000 +0000
@@ -3,7 +3,7 @@
 
 define(`confREQUIRE_LIBSM', `true')
 define(`confREQUIRE_SM_OS_H', `true')
-bldPRODUCT_START(`executable', `sendmail')
+bldPRODUCT_START(`executable', `sendmail_sendmail')
 define(`bldBIN_TYPE', `G')
 define(`bldINSTALL_DIR', `')
 define(`bldSOURCES', `main.c alias.c arpadate.c bf.c collect.c conf.c control.c convtime.c daemon.c deliver.c domain.c envelope.c err.c headers.c macro.c map.c mci.c milter.c mime.c parseaddr.c queue.c ratectrl.c readcf.c recipient.c sasl.c savemail.c sfsasl.c shmticklib.c sm_resolve.c srvrsmtp.c stab.c stats.c sysexits.c timers.c tls.c trace.c udb.c usersmtp.c util.c version.c ')
@@ -16,7 +16,7 @@
 define(`bldTARGET_INST_DEP', ifdef(`confINST_DEP', `confINST_DEP',
 `${DESTDIR}/etc/mail/submit.cf ${DESTDIR}${MSPQ}'))dnl
 define(`bldTARGET_LINKS', ifdef(`confLINKS', `confLINKS',
-`${DESTDIR}${UBINDIR}/newaliases ${DESTDIR}${UBINDIR}/mailq ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
+`${DESTDIR}${UBINDIR}/newaliases_sendmail ${DESTDIR}${UBINDIR}/mailq_sendmail ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
 )dnl
 
 # location of sendmail statistics file (usually /etc/mail/ or /var/log)
@@ -74,7 +74,7 @@
 		${LN} ${LNOPTS} ${M`'BINDIR}/sendmail $$i; \
 	done
 
-define(`confMTA_LINKS', `${DESTDIR}${UBINDIR}/newaliases ${DESTDIR}${UBINDIR}/mailq ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
+define(`confMTA_LINKS', `${DESTDIR}${UBINDIR}/newaliases_sendmail ${DESTDIR}${UBINDIR}/mailq_sendmail ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
 install-sm-mta: bldCURRENT_PRODUCT
 	${INSTALL} -c -o ${M`'BINOWN} -g ${M`'BINGRP} -m ${M`'BINMODE} bldCURRENT_PRODUCT ${DESTDIR}${M`'BINDIR}/sm-mta
 	for i in confMTA_LINKS; do \
